tests/conftest.py

```diff
@@ -192,6 +192,27 @@ def recipe_migration_cfep9(config_yaml, request):
     )
 
 
+@pytest.fixture(scope="function")
+def recipe_migration_cfep9_downgrade(config_yaml, recipe_migration_cfep9):
+    # write a downgrade migrator that lives next to the current migrator.
+    # Only this, more recent migrator should apply.
```

Comment:
the way migrators work by default is that the final version will be 1000 because migrators by default maximize versions.  This way when we move conda-forge pinning to `2000` we don't artificially make that lower by running the migrator